Automattic/woocommerce-payments
# Add Transactions > Uncaptured view with mocked data

Fixes #4501 

#### Changes proposed in this Pull Request

This adds an uncaptured view with mocked data and is disabled by default (behind a simple flag).










#### Testing instructions





* Get this PR
* Go to `client/transactions/index.tsx` and turn `displayAuthorizations` to `true`.
* Build the app and go to Transaction page
* You should see a new "Uncaptured" tab

-------------------


**Post merge**
